test/helpers/helpers.go

```diff
@@ -150,6 +151,19 @@ func NewMachineConfigPool(name string, mcSelector, nodeSelector *metav1.LabelSel
 	}
 }
 
+// NewIgnFile returns a simple ignition3 file from just path and file contents
+func NewIgnFile(path, contents string) ign3types.File {
```

Comment:
@cheesesashimi should I move this to `util.go`? Or should the other CreateIgnFile functions be moved to `helpers.go`?

Reply:
Also do you prefer CreateIgn3 to NewIgn?

Reply:
I'm fine with either CreateIgn3 or NewIgn3.

In terms of these helper funcs that create either an Ignition or MachineConfig, I think we should create another file called `test/helpers/fixtures.go` and put them in there.

Reply:
Okay may do that in a separate PR